django/core/models.py

```diff
@@ -57,7 +57,14 @@ def get_group(self):
         if _group is None:
             _group = self.group = Group.objects.get(name=self.value)
         return _group
-
+    
+    # can be called like #ComsesGroups.ADMIN.is_group_member(user)
+    def is_group_member(self, user):
+        for member in self.users():
+            if member.username == user.username:
+                return True
```

Comment:
IMO the instance method approach (as opposed to multiple static methods) is fine, though this implementation is less efficient than the direct query (`user.groups.filter(name=self.value).exists()`) that its meant to act as a convenience method for

Reply:
Yep, also @ldwiyono make sure your git commits follow the conventional commits guidelines:

https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
Yes, sorry, that is inefficient. I will try and push the more efficient `return self.users().filter(username=user.username).exists()`.

Reply:
I stuck with the static methods approach for now, just pushed a commit with some adjustments to the boolean logic in view_helpers.py as well